modules/base/tool.go

```diff
@@ -603,3 +606,21 @@ func EntryIcon(entry *git.TreeEntry) string {
 
 	return "file-text"
 }
+
+// SetupGiteaRoot Sets GITEA_ROOT if it is not already set and returns the value
+func SetupGiteaRoot() string {
+	giteaRoot := os.Getenv("GITEA_ROOT")
+	if giteaRoot == "" {
+		goPath := os.Getenv("GOPATH")
```

Comment:
Why can't we just use

```suggestion
		var err error
		giteaRoot, err = os.Getwd()
		if err != nil {
			_, filename, _ := runtime.Caller(0)
			giteaRoot = strings.TrimSuffix(filename, "modules/base/tool.go")
		}
```

Reply:
Made this change.